Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -71,12 +71,32 @@ class SimplifiedLoginViewController: UIViewController {
         return view
     }()
     
+    func setupNavigationBar(){
+        
+        let navigationBar = navigationController?.navigationBar
+        navigationBar?.topItem?.title = "Continue to log in"
```

Comment:
Add TODO mark to remeber about localization :)

Reply:
I've added it!